pyBSDate/wrappers.py

```diff
@@ -42,10 +43,11 @@ def isoformat(self, lang='en'):
         return self.strftime(u"%Y-%m-%d", lang)
 
     def timetuple(self):
-        raise NotImplementedError
+        yday = self.toordinal() - datetime.date(self.year, 1, 1).toordinal() + 1
+        return time.struct_time((self.year, self.month, self.day, 0, 0, 0, self.weekday(), yday, -1))
```

Comment:
the `year`, `month`, `day`, and `yday` should be in reference to bs date (`bs_year`, `bs_month`, `bs_day`, `bs_yday -> num days since Baisakh 01 this year`)